From 1a66d66ed98b3735b46d84d8b6f4de17e19b72a4 Mon Sep 17 00:00:00 2001
From: Darren Etheridge <detheridge@ti.com>
Date: Fri, 22 Mar 2013 15:47:50 -0500
Subject: [PATCH 143/177] tilcdc added some extra debug, and softened the
 wording of the debug messages when deciding if a mode is supported or not

---
 drivers/gpu/drm/tilcdc/tilcdc_crtc.c | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/tilcdc/tilcdc_crtc.c b/drivers/gpu/drm/tilcdc/tilcdc_crtc.c
index 63aec17..78965d1 100644
--- a/drivers/gpu/drm/tilcdc/tilcdc_crtc.c
+++ b/drivers/gpu/drm/tilcdc/tilcdc_crtc.c
@@ -445,13 +445,14 @@ int tilcdc_crtc_mode_valid(struct drm_crtc *crtc, struct drm_display_mode *mode,
 	/* some devices have a maximum allowed pixel clock */
 	/* configured from the DT */
 	if(mode->clock > priv->max_pixelclock) {
-	  DBG("Rejecting mode, pixel clock too high");
+	  DBG("Pruning mode, pixel clock too high");
 	  return MODE_BAD;
 	}
 
 	/* some devices further limit the max horizontal resolution */
 	/* configured from the DT */
 	if(mode->hdisplay > priv->max_width) {
+	  DBG("Pruning mode, above max width of %d supported by device", priv->max_width);
 	  return MODE_BAD;
 	}
 
@@ -459,7 +460,7 @@ int tilcdc_crtc_mode_valid(struct drm_crtc *crtc, struct drm_display_mode *mode,
 	/* configured from the DT */
 	bandwidth = mode->hdisplay * mode->vdisplay * drm_mode_vrefresh(mode);
 	if (bandwidth > priv->max_bandwidth) {
-	  DBG("Rejecting mode, exceeds defined bandwidth limit");
+	  DBG("Pruning mode, exceeds defined bandwidth limit");
 	  return MODE_BAD;
 	}
 
@@ -471,7 +472,7 @@ int tilcdc_crtc_mode_valid(struct drm_crtc *crtc, struct drm_display_mode *mode,
 		       (mode->hsync_end - mode->hsync_start == 32) &&
 		       (mode->vsync_start - mode->vdisplay == 3);
 		if (!rb) {
-			DBG("Throwing away because we only support reduced blanking");
+			DBG("Pruning mode, only support reduced blanking modes");
 			return MODE_BAD;
 		}
 	}
-- 
1.8.1.4

